config/sites/its.uiowa.edu/config_split.config_split.site.yml

```diff
@@ -10,7 +10,7 @@ stackable: true
 storage: folder
 folder: ../config/sites/its.uiowa.edu
 module:
-  its_core: 0
+  its_core: 10
```

Comment:
I think for ordering of field preprocessing, but there have been a lot of changes in what's changed and how, so it may not be necessary any more...I wish I had a better recollection of precisely what needed to happen in order.

Reply:
thanks, i am fine with keeping it. hopefully we don't need to revisit it again.